modules/exploits/multi/http/atutor_sqli.rb

```diff
@@ -243,7 +243,7 @@ def get_ascii_value(sql, cookie)
        mid = (lower + upper) / 2
        sqli = "#{sql}>#{mid}"
        result = perform_request(sqli, cookie)
-       if result =~ /There are \d entries./
+       if result =~ /There are \d{0,8} entries\./
```

Comment:
@wvu-r7 I would say since ```\d+``` is more generic let's go with it. Unless there is a 0.00000001% performance improvement with ```\d{1,8}``` :)